### ISSUE TITLE
[bug] [win10] Crash after running

### ISSUE THREAD

**Issue by cn-kali-team** [tauri-apps/tauri#6635]
### Describe the bug

My public repo [notion-rss](https://github.com/cn-kali-team/notion-rss) download https://github.com/cn-kali-team/notion-rss/releases
I have downloaded and installed the latest **MicrosoftEdgeWebView2RuntimeInstallerX64** on the official Microsoft website.

```
C:\Program Files\notion-rss>systeminfo

主机名:           DESKTOP-LA6R7JI
OS 名称:          Microsoft Windows 10 专业版
OS 版本:          10.0.10240 暂缺 Build 10240
OS 制造商:        Microsoft Corporation
OS 配置:          独立工作站
OS 构件类型:      Multiprocessor Free
注册的所有人:     Windows 用户
注册的组织:
产品 ID:          00330-80000-00000-AA993
初始安装日期:     2023/4/4, 20:12:05
系统启动时间:     2023/4/4, 21:19:57
系统制造商:       VMware, Inc.
系统型号:         VMware20,1
系统类型:         x64-based PC
处理器:           安装了 2 个处理器。
                  [01]: Intel64 Family 6 Model 158 Stepping 9 GenuineIntel ~3600 Mhz
                  [02]: Intel64 Family 6 Model 158 Stepping 9 GenuineIntel ~3600 Mhz
BIOS 版本:        VMware, Inc. VMW201.00V.20904234.B64.2212051119, 2022/12/5
Windows 目录:     C:\Windows
系统目录:         C:\Windows\system32
启动设备:         \Device\HarddiskVolume1
系统区域设置:     zh-cn;中文(中国)
输入法区域设置:   zh-cn;中文(中国)
时区:             (UTC+08:00)北京，重庆，香港特别行政区，乌鲁木齐
物理内存总量:     2,047 MB
可用的物理内存:   873 MB
虚拟内存: 最大值: 3,199 MB
虚拟内存: 可用:   1,974 MB
虚拟内存: 使用中: 1,225 MB
页面文件位置:     C:\pagefile.sys
域:               WORKGROUP
登录服务器:       \\DESKTOP-LA6R7JI
修补程序:         安装了 4 个修补程序。
                  [01]: KB3116097
                  [02]: KB3118714
                  [03]: KB3119598
                  [04]: KB3122962
网卡:             安装了 1 个 NIC。
                  [01]: Intel(R) 82574L Gigabit Network Connection
                      连接名:      Ethernet0
                      启用 DHCP:   是
                      DHCP 服务器: 192.168.179.254
                      IP 地址
                        [01]: 192.168.179.132
                        [02]: fe80::9094:3aa6:5454:ffa0
Hyper-V 要求:     已检测到虚拟机监控程序。将不显示 Hyper-V 所需的功能。

C:\Program Files\notion-rss>
```
- GUI Window Event Information
```xml
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="49152">1005</EventID> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2023-04-04T13:23:44.000000000Z" /> 
  <EventRecordID>300</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>DESKTOP-LA6R7JI</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data /> 
  <Data>notion-rss</Data> 
  <Data>00000000</Data> 
  <Data>0</Data> 
  </EventData>
  </Event>
```

- Cli Window Event Information
```xml
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Error" /> 
  <EventID Qualifiers="0">1000</EventID> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2023-04-04T13:23:44.000000000Z" /> 
  <EventRecordID>299</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>DESKTOP-LA6R7JI</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>notion-rss-cli.exe</Data> 
  <Data>0.1.0.0</Data> 
  <Data>642ae5ed</Data> 
  <Data>notion-rss-cli.exe</Data> 
  <Data>0.1.0.0</Data> 
  <Data>642ae5ed</Data> 
  <Data>c000001d</Data> 
  <Data>000000000024c698</Data> 
  <Data>107c</Data> 
  <Data>01d966f8adedb42d</Data> 
  <Data>C:\Program Files\notion-rss\notion-rss-cli.exe</Data> 
  <Data>C:\Program Files\notion-rss\notion-rss-cli.exe</Data> 
  <Data>1b18a66c-8db4-442d-8514-7e6ede459684</Data> 
  <Data /> 
  <Data /> 
  </EventData>
  </Event>
```

- But it works fine on my other Windows 10

```
λ systeminfo

主机名:           W10-11
OS 名称:          Microsoft Windows 10 企业版 LTSC
OS 版本:          10.0.17763 暂缺 Build 17763
OS 制造商:        Microsoft Corporation
OS 配置:          成员工作站
OS 构建类型:      Multiprocessor Free
注册的所有人:     xxxxx
注册的组织:
产品 ID:          xx
初始安装日期:     2021/5/25, 15:39:14
系统启动时间:     2023/2/23, 17:17:06
系统制造商:       VMware, Inc.
系统型号:         VMware Virtual Platform
系统类型:         x64-based PC
处理器:           安装了 2 个处理器。
                  [01]: Intel64 Family 6 Model 85 Stepping 7 GenuineIntel ~2993 Mhz
                  [02]: Intel64 Family 6 Model 85 Stepping 7 GenuineIntel ~2993 Mhz
BIOS 版本:        Phoenix Technologies LTD 6.00, 2020/11/12
Windows 目录:     C:\Windows
系统目录:         C:\Windows\system32
启动设备:         \Device\HarddiskVolume1
系统区域设置:     zh-cn;中文(中国)
输入法区域设置:   zh-cn;中文(中国)
时区:             (UTC+08:00) 北京，重庆，香港特别行政区，乌鲁木齐
物理内存总量:     16,383 MB
可用的物理内存:   7,569 MB
虚拟内存: 最大值: 20,457 MB
虚拟内存: 可用:   2,865 MB
虚拟内存: 使用中: 17,592 MB
页面文件位置:     C:\pagefile.sys
域:               xxxx
登录服务器:       \\xxxx
修补程序:         安装了 23 个修补程序。
                  [01]: KB5022511
                  [02]: KB4470788
                  [03]: KB4487038
                  [04]: KB4535680
                  [05]: KB4577586
                  [06]: KB4589208
                  [07]: KB5001404
                  [08]: KB5003243
                  [09]: KB5003711
                  [10]: KB5005112
                  [11]: KB5012170
                  [12]: KB5023702
                  [13]: KB5006754
                  [14]: KB5008287
                  [15]: KB5009642
                  [16]: KB5011574
                  [17]: KB5012675
                  [18]: KB5014031
                  [19]: KB5014797
                  [20]: KB5015896
                  [21]: KB5020374
                  [22]: KB5023789
                  [23]: KB5005701
网卡:             安装了 5 个 NIC。
                  [01]: vmxnet3 以太网适配器
                      连接名:      Ethernet0
                      启用 DHCP:   是
                      DHCP 服务器: xxx
                      IP 地址
                        [01]: xxx
                        [02]: xxx
                  [02]: VMware Virtual Ethernet Adapter for VMnet1
                      连接名:      VMware Network Adapter VMnet1
                      启用 DHCP:   否
                      IP 地址
                        [01]: xxx
                        [02]: xxx
                  [03]: VMware Virtual Ethernet Adapter for VMnet8
                      连接名:      VMware Network Adapter VMnet8
                      启用 DHCP:   否
                      IP 地址
                        [01]: xxx
                        [02]: xxx
                  [04]: Hyper-V Virtual Ethernet Adapter
                      连接名:      vEthernet (Default Switch)
                      启用 DHCP:   否
                      IP 地址
                        [01]: xxx
                        [02]: xxx
                  [05]: Virtual Network Adapter
                      连接名:      本地连接
                      状态:        媒体连接已中断
Hyper-V 要求:     已检测到虚拟机监控程序。将不显示 Hyper-V 所需的功能。
```
- it works
```
λ notion-rss-cli.exe

███╗   ██╗ ██████╗ ████████╗██╗ ██████╗ ███╗   ██╗      ██████╗ ███████╗███████╗
████╗  ██║██╔═══██╗╚══██╔══╝██║██╔═══██╗████╗  ██║      ██╔══██╗██╔════╝██╔════╝
██╔██╗ ██║██║   ██║   ██║   ██║██║   ██║██╔██╗ ██║█████╗██████╔╝███████╗███████╗
██║╚██╗██║██║   ██║   ██║   ██║██║   ██║██║╚██╗██║╚════╝██╔══██╗╚════██║╚════██║
██║ ╚████║╚██████╔╝   ██║   ██║╚██████╔╝██║ ╚████║      ██║  ██║███████║███████║
╚═╝  ╚═══╝ ╚═════╝    ╚═╝   ╚═╝ ╚═════╝ ╚═╝  ╚═══╝      ╚═╝  ╚═╝╚══════╝╚══════╝
Build your own RSS Feeds in Notion.
________________________________________________
:  https://github.com/cn-kali-team/notion-rss  :
:  https://blog.kali-team.cn/donate            :
 -----------------------------------------------

```

### Reproduction

1. Install window10 on the virtual machine 
2.  download the latest version of MSI installation for [notion-rss.](https://github.com/cn-kali-team/notion-rss/releases)
3.  After installation
4.  double-click and there is no response

### Expected behavior

work

### Platform and versions

```text
➜  notion-rss git:(main) ✗ cargo tauri info

Environment
  › OS: Arch Linux Rolling Release X64
  › Node.js: 19.8.1
  › npm: 8.19.2
  › pnpm: Not installed!
  › yarn: 1.22.19
  › rustup: 1.25.2
  › rustc: 1.67.1
  › cargo: 1.67.1
  › Rust toolchain: stable-x86_64-unknown-linux-gnu 

Packages
  › @tauri-apps/cli [NPM]: 1.2.3
  › @tauri-apps/api [NPM]: 1.2.0
  › tauri [RUST]: 1.2.4,
  › tauri-build [RUST]: 1.2.1,
  › tao [RUST]: 0.15.8,
  › wry [RUST]: 0.23.4,

App
  › build-type: bundle
  › CSP: script-src 'unsafe-eval' 'self'; default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self'; img-src data: 'self';
  › distDir: ../dist
  › devPath: http://localhost:1420/
  › framework: Vue.js
  › bundler: Vite

App directory structure
  ├─ node_modules
  ├─ src
  ├─ .cargo
  ├─ .idea
  ├─ public
  ├─ .github
  ├─ dev
  ├─ target
  ├─ src-tauri
  ├─ .git
  └─ dist
```

**Comment by amrbashir** [comment]
did you install [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-eg/download/details.aspx?id=48145) on the fresh new VM?

**Comment by cn-kali-team** [comment]
Yes, my new virtual machine already has Visual C++ Redistributable for Visual Studio 2015  installed.
Because when I downloaded and installed the link you provided, the operating system prompted me that I had already installed the updated version.
In fact, both of my virtual machines have this dependency installed.

Even more strangely, my first version worked properly on two virtual machines
https://github.com/cn-kali-team/notion-rss/releases/tag/v2023.3.11
